Tutorials/Sprite-Related-Tutorials/Getting Started With Sprites in Splashkit Tutorial - C#/Getting Started With Sprites in Splashkit Tutorial - CSharp.mdx

```diff
@@ -23,47 +23,85 @@ don't interact with those.
 In order to be able to draw an image to a screen, we first have to provide some information to
 Splashkit. We have to have Bitmap images to pass into our
 [Create Sprite](https://splashkit.io/api/sprites/#create-sprite) function so that it can create a
-workable _Sprite_ file type for us. This looks something like this inside our Main() method:
-  
-```bash
-Private Bitmap happySun;
-Private Sprite happySunSprite;
-```
+workable _Sprite_ file type for us.
 
 Line by line, this can be broken down as:
 
 1. Instantiate a new Object happySun, of Type [Bitmap](https://splashkit.io/api/types/#bitmap)
 
-```csharp
+<Tabs>
+  <TabItem label="C#">
+  ```csharp
 Bitmap happySun;
+Sprite happySunSprite;
```

Comment:
You should probably remove Sprite happySunSprite as it's created later on in step 2.